File: IN1010 - Object Oriented programming/Game of Life GUI MVC/Verden.java
class Verden{
    Rutenett rutenett;
    int genNr;
    public Verden(int rader, int kolonner){
        rutenett = new Rutenett(rader, kolonner);
        genNr = 0; 
        rutenett.fyllMedTilfeldigeCeller();
        rutenett.kobleAlleceller();
    }

    public void tegn(){
        rutenett.tegnRutenett();
        System.out.println("Generasjons nummer:" + this.genNr);
        System.out.println("Antall levende celler:"+rutenett.antallLevende());
    }

    public  Rutenett hentRutenett(){
        return rutenett;
    }

    public int hentAntallLevende(){
        return rutenett.antallLevende();
    }

    public void oppdatering(){
        rutenett.oppdaterCeller();
        genNr ++;

    }
}

File: IN1000 - Introduction to object oriented programming/Game of Life - Python/verden.py
from rutenett import Rutenett
class Verden:
    def __init__(self, rader, kolonner):
        self._rutenett = Rutenett(rader, kolonner)
        self._generasjonsnummer = 0
        self._rutenett.fyll_med_tilfeldige_celler()
        self._rutenett.koble_celler()

    def tegn(self):
        self._rutenett.tegn_rutenett()
        print("")
        print(f"Dette er {self._generasjonsnummer}. generasjon.")
        print(f"Det er {self._rutenett.antall_levende()} levende celler.")

    def oppdatering(self):
        for celle in self._rutenett.hent_alle_celler():
            celle.tell_levende_naboer()
        for celle in self._rutenett.hent_alle_celler():
            celle.oppdater_status()
        self._generasjonsnummer += 1
